standard-worker-container/pom.xml

```diff
@@ -34,7 +34,7 @@
     <dependencies>
         <dependency>
             <groupId>com.github.cafapi.codec</groupId>
-            <artifactId>codec-json</artifactId>
```

Comment:
Obviously we can't actually merge this into `develop` as it would be a breaking change.

I've created a developer build for it:
http://sou-jenkins2.hpeswlab.net/job/WorkerFramework/view/Developer/